vector/src/main/java/im/vector/app/features/signout/soft/SoftLogoutActivity.kt

```diff
@@ -73,7 +73,10 @@ class SoftLogoutActivity : LoginActivity() {
                 )
             }
             is SoftLogoutViewEvents.ClearData        -> {
-                MainActivity.restartApp(this, MainActivityArgs(clearCredentials = true))
+                MainActivity.restartApp(this, MainActivityArgs(
+                        clearCredentials = true,
+                        isUserLoggedOut = true
```

Comment:
I think from SoftLogout if we are clearing the data we should specify that the user is now logged out, as the text in softlogout view suggests will happen:
> Clear it if you’re finished using this device, or want to sign in to another account.

 Otherwise we try and send the logout request which will always fail(as we no longer have a valid access token.). I have seen the app get into a weird state where an alert keeps prompting us if we would like to retry the logout request and it becomes difficult to get out of the app.

Reply:
OK